src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -18,13 +18,41 @@
 #include "Utilities/Gsl.hpp"
 
 namespace CurvedScalarWave::Worldtube::Tags {
-template <size_t Dim>
-void CenteredFaceCoordinatesCompute<Dim>::function(
-    const gsl::not_null<std::optional<tnsr::I<DataVector, Dim, Frame::Grid>>*>
+template <size_t Dim, typename Frame, bool Centered>
+void FaceCoordinatesCompute<Dim, Frame, Centered>::function(
+    const gsl::not_null<std::optional<tnsr::I<DataVector, Dim, Frame>>*> result,
+    const ::ExcisionSphere<Dim>& excision_sphere, const Element<Dim>& element,
+    const tnsr::I<DataVector, Dim, Frame>& coords, const Mesh<Dim>& mesh) {
+  const auto direction = excision_sphere.abutting_direction(element.id());
+  if (direction.has_value()) {
+    const size_t grid_size =
+        mesh.slice_away(direction->dimension()).number_of_grid_points();
+    if (result->has_value()) {
+      destructive_resize_components(make_not_null(&(result->value())),
+                                    grid_size);
+    } else {
+      result->emplace(grid_size);
+    }
+    evolution::dg::project_tensor_to_boundary(make_not_null(&(result->value())),
+                                              coords, mesh, direction.value());
```

Comment:
I forgot about this in the previous PR: this carries an interpolation error on Legendre-Gauss meshes. If you want to support LG meshes it would be better to use `interface_logical_coordinates` (from `Domain/InterfaceLogicalCoordinates.hpp`) and use coordinate maps to go grid/inertial coords. If you don't need LG meshes it would be safer to assert the quadrature and use `data_on_slice` from `DataStructures/Tensor/Slice.hpp` (and `index_to_slice_at` from `Domain/Structure/`).

Reply:
I don't really need to support Gauss points. But from what I can tell, exactly the same thing is done in `InternalMortarDataImpl.hpp` for both quadratures at each internal boundary every time step so there should not be any additional error. `project_tensor_to_boundary` essentially just mimics `data_on_slice` for GaussLobatto points.

Reply:
We have no choice but to project (interpolate) tensor data to the face for Gauss points, but coordinates can be evaluated directly on the face because we know them analytically. Add a comment at least, or disable Gauss points as mentioned above if you don't need them.

Reply:
ok, I have disabled Gauss now